### ISSUE TITLE
Unread and important mail widgets are showing deleted and junk emails

### ISSUE THREAD

**Issue by jakobroehrl** [nextcloud/mail#8849]
I'm not sure but I think earlier there were only unread/important inbox mails shown up, now all mails are shown
In both of this dashlets:
![grafik](https://github.com/nextcloud/mail/assets/24530680/070a1a73-d655-4bce-92e7-bb4cbd331c18)

### Expected behavior
only unread/important inbox mails

**Comment by GretaD** [comment]
Hello,

that is not a bug, that is a feature. If you dont want the unread mail to show up on your dashboard, click customize and remove the widgets that you dont want to be shown.

**Comment by ChristophWurst** [comment]
I think the report is about the messages in the widgets and that read and not important messages show

**Comment by GretaD** [comment]
Ah, sorry, because the screenshot has only the widget title, i thought they were wondering about it. I cannot reproduce this one though

**Comment by ChristophWurst** [comment]
Checked my unread widget. It only shows messages that are unread. If I read one of them it vanishes from the dashboard after page reload.

**Comment by max65482** [comment]
For me the "Unread Mail" feature suddenly started to include the junk mail folder.
So NC welcoms me with my latest unread spam mails...
(server: 27.1.0, mail: 3.4.4)

**Comment by Sanookmakmak** [comment]
No, the mail dashboard does not exclude the junk folder when checking for unread messages.

**Comment by st3iny** [comment]
Confirmed. Unread messages from the trash and junk mailboxes are shown in the unread emails widget.

**Comment by oli-ver** [comment]
@jakobroehrl @kesselb Does this fix work for you? In my case mails from the spam folder are still visualized on the widget.

**Comment by ChristophWurst** [comment]
Thank you for you feedback.

The fix is not released.

**Comment by oli-ver** [comment]
@ChristophWurst thanks for the fast feedback. In what release will this be included?

**Comment by ChristophWurst** [comment]
v5.0.7

**Comment by oli-ver** [comment]
Yeah, sorry just saw that it was included in 5.0.7. Thanks so much, working now!
